packages/client/src/views/SysAdmin/Team/user/userProfilie/UserProfile.tsx

```diff
@@ -384,7 +384,7 @@ class UserProfileComponent extends React.Component<Props, State> {
                           id="office-link"
                           onClick={() =>
                             this.props.goToTeamUserList(
-                              user.primaryOffice as ISearchLocation
+                              (user.primaryOffice as ISearchLocation).id
```

Comment:
I wonder if this would be clearer just as 

```ts
this.props.goToTeamUserList(user.primaryOffice!.id)
```
what do you think?

Reply:
Yes, I think that will be better .